.travis.yml

```diff
@@ -15,4 +28,65 @@ before_script:
   - eval "$(ssh-agent -s)"
   - ssh-add .ssh-keys/travis
 
-script: true
+script:
+  # Current dir and temporary dir for storing new packaged charts and index files
+  - set -e
+  - REPO_DIR=/home/travis/build/kubenow/helm-charts
+  - BUILD_DIR=$(mktemp -d)
+
+  # Useful URLs
+  - REPO_URL=`git config remote.origin.url`
+  - SSH_REPO=${REPO_URL/https:\/\/github.com\//git@github.com:}
+  - SHA=`git rev-parse --verify HEAD`
+
+  # Push temporary directory to the stack
+  - pushd $BUILD_DIR
+
+  # Iterate over all charts are package them
+  - for dir in `ls ${REPO_DIR}/charts`; do
```

Comment:
please write it like we do here: https://github.com/kubenow/KubeNow/blob/master/.travis.yml#L140-L149